lib/src/integrationtest.rs

```diff
@@ -101,6 +101,27 @@ where
     Ok(())
 }
 
+async fn test_pull_containers_storage() -> Result<()> {
+    let fixture = crate::fixture::Fixture::new_v1()?;
+    let srcpath = fixture.path.join("src/repo");
+    let srcrepo_parsed = crate::cli::parse_repo(&srcpath).unwrap();
+    let ir = crate::cli::parse_imgref(
+        "ostree-unverified-image:containers-storage:quay.io/fedora/fedora-coreos:testing-devel",
```

Comment:
This assumes this image has already been fetched, but we'd need to do that explicitly in the test flow right?

I'm also realizing that we probably want to split out a different kind of integration test...this one as you've noted assumes it's already running in a container image.

It may be simpler to do one in shell script.  Hmm, we already have `ci/priv-integration.sh` and ISTM in theory we could potentially just extend that?  It might be though that we'd need to bind mount some other things from the host to make it work (e.g. `/var/lib/containers`).  Nested container stuff is messy...I guess we could also try to have a CI flow that builds natively for Ubuntu, but that brings its own mess.

The other alternative is to use virtualization for this, i.e. we add Prow or CoreOSCI to this repo.  Which is probably overdue.